tensornetwork/contractors/opt_einsum_paths/path_contractors.py

```diff
@@ -82,14 +133,15 @@ def branch(net: network.TensorNetwork, memory_limit: Optional[int] = None,
   Returns:
     The network after full contraction.
   """
-  alg = functools.partial(opt_einsum.paths.branch, memory_limit=memory_limit,
-                          nbranch=nbranch)
-  return base(net, alg)
+    alg = functools.partial(
```

Comment:
Please keep all formatting to tab size 2. (I don't know why our linter isn't catching this).